```
       > In file included from src/utils/folderutils.cpp:39:
       > In file included from /nix/store/vqqyd2l7ncbw3m5dlrcyndfl6984w7z6-quazip-1.5-dev/include/QuaZip-Qt6-1.5/quazip/quazip.h:30:
       > /nix/store/vqqyd2l7ncbw3m5dlrcyndfl6984w7z6-quazip-1.5-dev/include/QuaZip-Qt6-1.5/quazip/quazip_qt_compat.h:18:12: fatal error: 'QtCore5Compat/QTextCodec' file not found
       >    18 | #  include <QtCore5Compat/QTextCodec>
       >       |            ^~~~~~~~~~~~~~~~~~~~~~~~~~
       > 1 error generated.
       > make[1]: *** [Makefile.Release:17770: ../release64/folderutils.o] Error 1
       > make[1]: *** Waiting for unfinished jobs....
       > make[1]: Leaving directory '/private/tmp/nix-build-fritzing-1.0.5.drv-0/source'
       > make: *** [Makefile:49: release] Error 2
       For full logs, run 'nix log /nix/store/28brxm47s6v4dbac4qlmqd8qh53v7r9w-fritzing-1.0.5.drv'.
```

Quazip <-> Qt5 compat issues?

--- a/pkgs/applications/science/electronics/fritzing/default.nix
+++ b/pkgs/applications/science/electronics/fritzing/default.nix
@@ -100,11 +100,14 @@ stdenv.mkDerivation {
   '';
 
   env = {
-    NIX_CFLAGS_COMPILE = lib.concatStringsSep " " [
-      "-I${lib.getDev quazip}/include/QuaZip-Qt${lib.versions.major qtbase.version}-${quazip.version}"
-      "-I${svgpp}/include"
-      "-I${clipper}/include/polyclipping"
-    ];
+    NIX_CFLAGS_COMPILE = lib.concatStringsSep " " (
+      [
+        "-I${lib.getDev quazip}/include/QuaZip-Qt${lib.versions.major qtbase.version}-${quazip.version}"
+        "-I${svgpp}/include"
+        "-I${clipper}/include/polyclipping"
+      ]
+      ++ lib.optionals stdenv.hostPlatform.isDarwin [ "-F${qt5compat}/lib" ]
+    );
     NIX_LDFLAGS = "-lquazip1-qt${lib.versions.major qtbase.version}";
   };
 

